lpr.Common/Interfaces/Services/IGitHubService.cs

```diff
@@ -1,11 +1,12 @@
-﻿using lpr.Common.Models;
+﻿using System.Threading.Tasks;
+using lpr.Common.Models;
 
 namespace lpr.Common.Interfaces.Services
 {
     public interface IGitHubService
     {
         Task<GithubUser> GetGithubUser(string token);
         Task<Account> GetRegisteredUser(int githubId);
-        Task<Account> Register(GithubUser githubUser);
+        Account Register(GithubUser githubUser);
```

Comment:
Shouldn't this be awaitable?

Reply:
This register function sets the GitHub in our database. So no outside HTTP request is made in the function.